Sources/TokamakLint/Reporters/TokamakLogHandler.swift

```diff
@@ -0,0 +1,101 @@
+//
+//  TokamakLogHandler.swift
+//  TokamakLint
+//
+//  Created by Matvii Hodovaniuk on 5/30/19.
+//
+
+import Foundation
+import Logging
+
+public struct LogFileCreationError: Error {
+  public init() {}
```

Comment:
this can be removed now as `LogFileCreationError` is no longer created outside of `TokamakLint` module